features/place/duplicate.feature

```diff
@@ -7,12 +7,12 @@ Feature: Test creating places
     And I send and accept "application/json"
 
   Scenario: Be prevented from creating a new place if we already have one on that address
-    Given I create a minimal place and save the "id" as "placeId" then I should get a "201" response code
-    Given I create a minimal place and save the "id" as "placeId" then I should get a "303" response code
-    Then the JSON response should be:
+    Given I prevent duplicate creation
+    Given I create a random name of 6 characters and keep it as "name"
```

Comment:
This call is no longer needed, the duplicate error comes from executing lines 12 and 13

Reply:
I don't think so, because the default place that is always created (den engel) will probably already exist because of a previous feature test. So I need to make sure the first place can be created.

Reply:
Ok, I made a mistake here because I did not find any usages of `place-with-required-fields-and-variable-name.json`